source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidTokenClient.cs

```diff
@@ -132,6 +133,41 @@ public void FetchTokens(bool silent, Action<int> callback)
             PlayGamesHelperObject.RunOnGameThread(() => DoFetchToken(silent, callback));
         }
 
+        public void RequestPermissions(Action<RequestPermissionStatusCodes> callback, string[] scopes)
+        {
+            using (var bridgeClass = new AndroidJavaClass(HelperFragmentClass))
+            using (var currentActivity = AndroidHelperFragment.GetActivity())
+            using (var task =
+                bridgeClass.CallStatic<AndroidJavaObject>("showRequestPermissionsUi", currentActivity, oauthScopes.Union(scopes).ToArray()))
+            {
+                AndroidTaskUtils.AddOnSuccessListener<AndroidJavaObject>(task, result =>
+                {
+                    var code = (RequestPermissionStatusCodes) result.Get<int>("status");
+                    if (code == RequestPermissionStatusCodes.Success)
+                    {
+                        account = result.Get<AndroidJavaObject>("account");
+                        oauthScopes = oauthScopes.Union(scopes).ToList();    
+                    }
```

Comment:
Should we build account with combined scopes?

Reply:
Yes, I missed putting it here. Now it's there in onActivityResult                 

GoogleSignInAccount account = GoogleSignIn.getAccountForScopes(helperFragment.getActivity(), scopes[0], scopes);